controllers/user.controller.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-underscore-dangle */
+const User = require('../models/user.model');
+
+// update user
+const updateUser = async (ctx, next) => {
+  try {
+    const user = await User.findByIdAndUpdate(
+      { _id: ctx.request.body._id },
+      ctx.request.body,
+      { new: true },
+    );
+    // send back updated user
+    ctx.body = user;
+  } catch (error) {
+    ctx.status = error.statusCode || error.status || 500;
+    ctx.body = { message: error.message };
+  }
+  await next();
```

Comment:
Since this is the controller, the final step of your chain, invoking next is not necessary, as you are setting the response body on line 13 to send in response the updated user.